src/backend/brokers/go.emails/email.go

```diff
@@ -52,6 +52,17 @@ func (b *EmailBroker) MarshalEmail(msg *Message) (em *EmailMessage, err error) {
 		},
 		Message: msg,
 	}
+	// sha256 internal message id to form external message id
```

Comment:
you could make use of `NewMessageId` func you created in the same pkg (and put this func within email.go instead of encrypt.go)

Reply:
done in #7525f865